components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -7,11 +7,16 @@ export const toProperCase = (value: string) =>
   value.replace(/\w\S*/g,
     (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase())
 
-export const isRemoteImageURL = (url?: string) =>
-  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/')
+export const isFromDifferentOrigin = (url?: string) =>
```

Comment:
the name is sort of misleading since a URL like `blob:...` would be from a different origin yet pass this check. maybe this can just return `url.startsWith('chrome://')`?

Reply:
actually i'm not sure in what case it's supposed to return false, but it seems like it should either be simplified or renamed